lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseTokenizer.java

```diff
@@ -219,9 +219,9 @@ public JapaneseTokenizer(UserDictionary userDictionary, boolean discardPunctuati
   }
 
   /**
-   * Create a new JapaneseTokenizer, supplying a custom system dictionary and unknown dictionary.
-   * <p>
-   * Uses the default AttributeFactory.
+   * <p>Create a new JapaneseTokenizer, supplying a custom system dictionary and unknown dictionary.
+   * This constructor provides an entry point for users that want to constructcustom language models
```

Comment:
Seems a whitespace is missing : constructcustom -> construct custom

Reply:
Thanks for the review; I'll fix and push.